docs/code/MixingITKAndNumPy.py

```diff
@@ -14,34 +14,40 @@
 # Run filters on itk.Image
 
 # View only of itk.Image, data is not copied
-np_view = itk.GetArrayViewFromImage(itk_image)
+np_view = itk.array_view_from_image(itk_image)
 
 # Copy of itk.Image, data is copied
-np_copy = itk.GetArrayFromImage(itk_image)
+np_copy = itk.array_from_image(itk_image)
 
 
-# Do numpy stuff
+# Do numpy stuff...
 
 
 # Convert back to itk, view only, data is not copied
-itk_np_view = itk.GetImageViewFromArray(np_copy)
+itk_np_view = itk.image_view_from_array(np_copy)
 
 # Convert back to itk, data is copied
-itk_np_copy = itk.GetImageFromArray(np_copy)
+itk_np_copy = itk.image_from_array(np_copy)
 
 # Save result
 itk.imwrite(itk_np_view, output_filename)
 
 # Vnl matrix from array
```

Comment:
The same comment applies to `Vnl` vs. `VNL`.

Reply:
All `VNL` now!